src/cli/cli.ts

```diff
@@ -97,6 +97,12 @@ export class Cli {
                     return options.protocol == 'https';
                 }
             }, {
+                name: 'ssPassphrase',
```

Comment:
Since this probably won't be a common configuration needed, I think we should remove it from the setup guide, as it will spark questions for the many who don't require one.